reverse-proxy.md

```diff
@@ -257,7 +257,7 @@ server {
     server_name <your-nc-domain>;
 
     location / {
-        resolver localhost;
+        resolver localhost; # Note: if you don't run a dns resolver on localhost:53, you should use another resolver
```

Comment:
Ah thar explains things!

Reply:
```suggestion
        resolver localhost; # Note: if you don't run a dns resolver on localhost:53, you should use another resolver. Otherwise you need to use 127.0.0.1 in the line below this one instead of localhost
```

Reply:
Or maybe add the note into the line directly? That if not resolver is defined you need to use 127.0.0.1?

Reply:
```suggestion
        resolver localhost; # Note: you need to set a valid dns resolver here or use 127.0.0.1 / [::1] instead of localhost in the line below
```

Reply:
Yeah looks good, or maybe add the advice in the line below?

Reply:
```suggestion
        resolver localhost; # Note: if you don't run a dns resolver on localhost:53, you should use another resolver. See https://stackoverflow.com/a/49642310 for a better explanation
```